requirements.txt

```diff
@@ -1,17 +1,18 @@
 py-solc
-web3
-eth-tester[py-evm]
+web3[tester]>=5.0.0b2
+eth-tester
+py-evm
```

Comment:
We dont use py-evm explicitly, so the one before should be again `eth-tester[py-evm]`

Reply:
Here my commit message from the last commit applies. If we have eth-tester in a requirements file, py-evm will not be installed.

Reply:
but that doesn't matter to us, because we don't have a plain `eth-tester` as requirement. In the web3 case it was different.